docs/guide/repository.md

```diff
@@ -484,209 +474,76 @@ ezpublish:
-| `namespace` | A namespace to prefix cache keys with to avoid key conflicts with other eZ Platform sites on same eZ Platform installation; default is `null`. |
-
 #### Redis
 
 This cache backend is using [Redis, a in-memory data structure store](http://redis.io/), via [Redis pecl extension](https://pecl.php.net/package/redis). This is an alternative cache solution for [multi server (cluster) setups](clustering.md), besides using Memcached.
 
-**Available settings**
+See [Redis Cache Adapter in Symfony documentation](https://symfony.com/doc/3.4/components/cache/adapters/redis_adapter.html#configure-the-connection)
+for information on how to configure Redis.
 
-|||
-|------|------|
-|`servers`|Array of Redis servers:</br>`server`: Host or IP of your Redis server</br>`port`: Port that Redis is listening to (default: 6379)</br>`ttl`: Optional float value of connection timeout in seconds</br>`socket`: Optional boolean value to specify if server refers to a socket (default: false)|
-|`password`|Optional setting if there is a password to connection to a given Redis server in plain text over the network.|
-|`database`|Optional setting to specify a given Redis database to use.|
+!!! note
+
+    To use this, you need to set `cache_service_name` to `cache.redis`.
 
 **Example**
 
 ``` yaml
-# config.yml example
-stash:
-    caches:
-        default:
-            drivers: [ Redis ]
-            Redis:
-                servers:
-                    -
-                        server: 'redis1.ez.no'
-                        port: 6379
-                    -
-                        server: 'redis2.ez.no'
-                        port: 6379
+services:
+    cache.redis:
+        parent: cache.adapter.redis
+        tags:
+            - name: cache.pool
+              clearer: cache.app_clearer
+              provider: 'redis://secret@example.com:1234/13'
 ```
 
-##### Memcache(d)
+#### Memcached
 
 This cache backend is using [Memcached, a distributed caching solution](http://memcached.org/). This is the main supported cache solution for [multi server (cluster) setups](clustering.md), besides using Redis.
 
-!!! note
+See [Memcached Cache Adapter in Symfony documentation](https://symfony.com/doc/3.4/components/cache/adapters/memcached_adapter.html#configure-the-connection)
+for information on how to configure Memcached.
 
-    Stash supports both the [php-memcache](http://php.net/memcache) and [php-memcached](http://php.net/memcached) extensions. **However** only php-memcached is officially supported as php-memcache is missing many features and is less stable.
+When using Memcached cache backend, you *may* use inMemory to reduce network traffic as long as you are aware of its limitations mentioned above. However you should disable in web servers where there is concurrency on updates, for instance on dedicated editorial server.
```

Comment:
> However you should disable in web servers where there is concurrency on updates, for instance on a dedicated editorial server.

What should you disable?

Reply:
Actually this is no longer relevant for 2.x, there is no inMemory option